### DISCUSSION TITLE
hainanu-course-comments/1.%E5%AD%A6%E9%95%BF%E5%AD%A6%E5%A7%90%E8%AF%B4/%E4%BF%9D%E7%A0%94%E7%BB%8F%E9%AA%8C%E5%88%86%E4%BA%AB/

### DISCUSSION THREAD

**Discussion by giscus** [beiyuouo/hainanu-course-comments#7] [General]
# hainanu-course-comments/1.%E5%AD%A6%E9%95%BF%E5%AD%A6%E5%A7%90%E8%AF%B4/%E4%BF%9D%E7%A0%94%E7%BB%8F%E9%AA%8C%E5%88%86%E4%BA%AB/

https://www.bj-yan.top/hainanu-course-comments/1.%E5%AD%A6%E9%95%BF%E5%AD%A6%E5%A7%90%E8%AF%B4/%E4%BF%9D%E7%A0%94%E7%BB%8F%E9%AA%8C%E5%88%86%E4%BA%AB/

**Comment by visions-ma** [comment]
菜鸡前来围观

**Reply by yyyanbj** [reply]
前排滋瓷马佬orz

**Comment by Arcticius** [comment]
膜

**Comment by shiyan14712** [comment]
膜！！！ 顺便前排兹磁！
